server/protocols/webadmin/webadmin-mailbox/src/test/java/org/apache/james/webadmin/routes/UserQuotaRoutesTest.java

```diff
@@ -420,6 +420,28 @@ public void getQuotaShouldReturnOnlySpecifiedValues() throws Exception {
         softly.assertThat(jsonPath.getObject("global." + COUNT, Long.class)).isNull();
     }
 
+    @Test
+    public void getQuotaShouldReturnGlobalValuesWhenNoUserValuesDefined() throws Exception {
+        maxQuotaManager.setDefaultMaxStorage(QuotaSize.size(1111));
+        maxQuotaManager.setDefaultMaxMessage(QuotaCount.count(12));
+
+        JsonPath jsonPath =
+            given()
+                .get(QUOTA_USERS + "/" + BOB.asString())
+                .then()
```

Comment:
Indent

Reply:
sorry about that, intellij doesn't like our indent and any copy/paste break it ):